examples/azure/apache_server.json

```diff
@@ -0,0 +1,55 @@
+{
+
+    "builders": [{
+
+      "type": "azure-arm",
+
+  
+
+      "client_id": "d8j04jc-03dd-466a-9745-1f5bcmj9yn5a30e",
+
+      "client_secret": "405c41b4-ddc5-4cc1-b473-ae458tu7f1",
+
+      "tenant_id": "aeefb57f-cec2-4491-bf00-2555re4d7b",
+
+      "subscription_id": "52646-f20d-4a7a-a75d-5446b2135a6",
+
```

Comment:
are these your secrets ?

Reply:
Just hiding them in case they are your actual secrets, please revert that commit if that's not the case.
```suggestion
```